src/test/regress/expected/multi_extension.out

```diff
@@ -1010,7 +1076,7 @@ FROM
 	pg_dist_node_metadata;
  partitioned_citus_table_exists_pre_11 | is_null
 ---------------------------------------------------------------------
-                                       | t
+ f                                     | f
```

Comment:
Note to reviewer:

Given that now we have a few tests that upgrade Citus to 13.2 before this line, 'partitioned_citus_table_exists_pre_11' field remains in the one and only row of the pg_dist_node_metadata as "false", which is fine.